packages/chai-jscodeshift/index.js

```diff
@@ -17,7 +17,7 @@ var config = {
 module.exports = function chaiJSCodeShift(options) {
   var finalConfig = merge({}, config, options || {});
 
-  // eslint-disable-next-line no-shadow, func-style
+  // eslint-disable-next-line no-shadow, func-style, func-name-matching
   var chaiHelper = function chaiJSCodeShift(chaiOptions) {
```

Comment:
Rename the function here to `chaiHelper` and we should be able to ditch the `no-shadow` and `func-name-matching` eslint disables.